CONTRIBUTORS.md

```diff
@@ -14,4 +14,4 @@ SpeziStorage contributors
 * [Paul Schmiedmayer](https://github.com/PSchmiedmayer)
 * [Vishnu Ravi](https://github.com/vishnuravi)
 * [Andreas Bauer](https://github.com/Supereg)
-* [Philipp Zagar]https://github.com/philippzagar)
+* [Philipp Zagar](https://github.com/philippzagar)
```

Comment:
Thank you! Please also add yourself here 🚀

Reply:
Thanks for the fix, and yes, you should definitely add yourself there! 🚀